nui_shared_utils/utils.py

```
@@ -18,113 +16,107 @@
+    if hasattr(config, "aws_region") and config.aws_region:
         return config.aws_region
-    
+
     # Check boto3 session default
     try:
+        import boto3
+
         session = boto3.session.Session()
         if session.region_name:
             return session.region_name
     except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)